Build fails on Windows

After following instructions to build source on windows (including onnx build), getting this error (tried on VS Code, VS2022). Same error when targeting standalone build or JUCE VST helper. Nuked build folder to no avail

C:\Users\ryanb\OneDrive\Documents\NeuralNote\NeuralNote\NeuralNote\Source\Components\NeuralNoteLNF.cpp(71,15): error C2065: 'LABEL_FONT': undeclared identifier [C:\Users\ryanb\OneDrive\Documents\NeuralNote\NeuralNote\build\NeuralNote.vcxproj]
[proc] The command: "C:\Program Files\CMake\bin\cmake.EXE" --build c:/Users/ryanb/OneDrive/Documents/NeuralNote/NeuralNote/build --config Debug --target ALL_BUILD -j 18 -- exited with code: 1
[driver] Build completed: 00:00:03.978
[build] Build finished with exit code 1

Heres some compiler ID from the CMake config output if that helps

[cmake] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.22631.
[cmake] -- The C compiler identification is MSVC 19.41.34120.0
[cmake] -- The CXX compiler identification is MSVC 19.41.34120.0

--- a/NeuralNote/Source/Components/NeuralNoteLNF.cpp
+++ b/NeuralNote/Source/Components/NeuralNoteLNF.cpp
@@ -68,7 +68,7 @@ void NeuralNoteLNF::drawTooltip(Graphics& g, const String& text, int width, int 
     g.fillRoundedRectangle(0.0f, 0.0f, static_cast<float>(width), static_cast<float>(height), 5.0f);
 
     g.setColour(BLACK);
-    g.setFont(LABEL_FONT);
+    g.setFont(UIDefines::LABEL_FONT());
 
     auto text_std = text.toStdString();
     int num_line_breaks = static_cast<int>(std::count(text_std.begin(), text_std.end(), '\n'));

